app/models.py

```diff
@@ -214,7 +214,7 @@ class classification(basemodel):
   contract_list = db.Column(db.ARRAY(db.Integer), unique=False, nullable=False, primary_key=False)
   identifiability_rule_list = db.Column(db.ARRAY(db.Integer), unique=False, nullable=False, primary_key=False)
   recipe_id = db.Column(db.Integer, unique=False, nullable=False, primary_key=False)
-  label = db.Column(db.String(60), unique=False, nullable=False, primary_key=False)
+  label = db.Column(db.String(1000), unique=False, nullable=False, primary_key=False)
```

Comment:
In future, might look into just using "text" types for these columns over "varchar", unless it's important that you restrict the lengths. No performance difference (see https://stackoverflow.com/questions/4848964/postgresql-difference-between-text-and-varchar-character-varying)

Reply:
I learned something today.  That was not always the case that text performed as well as varchar.  Guess I am operating off an old model of how databases work.  Thanks for the link- good info.